lib/core/presentation/pages/edit_profile/widgets/profile_textfield_widget.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:app/core/presentation/widgets/lemon_text_field.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:flutter/material.dart';
+
+class ProfileTextFieldWidget extends StatelessWidget {
```

Comment:
Same, why not using common text input widget ? and just enhance that component

If design too difference, we need ask designer to make consistent input

Reply:
In here I'm using LemonTextField which is a common widget for entire app. ProfileTextFieldWidget come with a label on top so I create a new class support this. Since all the text field in EditProfile need a label.